builtin/game/auth.lua

```diff
@@ -53,7 +52,7 @@ core.builtin_auth_handler = {
 			name = name,
 			password = password,
 			privileges = core.string_to_privs(core.settings:get("default_privs")),
-			last_login = os.time(),
+			last_login = -1,  -- Defer login time calculation until on_joinplayer
```

Comment:
It's much more obvious to point to the place where `last_login` is actually set:
```suggestion
			last_login = -1,  -- Defer login time calculation until record_login (called by on_joinplayer)
```